test/tests/functional/pbs_qmgr.py

```diff
@@ -0,0 +1,126 @@
+            # Spaces are prefix for new attribute lines
+            # Make sure that this is a new attribute line
+            self.assertTrue("=" in line)
+        elif line[0] == "\t":
+            # Tabs are prefix for line extensions
+            # Make sure that this is a line extension
+            self.assertTrue("=" not in line)
+
+    def test_listcmd_whitespaces(self):
+        """
+        Check that the prefix for new attributes listed out by qmgr list
+        are spaces and that for line extensions is a tab
+        """
+        fd, fn = self.du.mkstemp()
+        node_prefix = "vn"
+        nodename = node_prefix + "[0]"
+        vndef_file = None
+        try:
+            # Check 1: New attributes are prefixed with spaces and not tabs
+            # Execute qmgr -c 'list sched' and store output in a temp file
+            if self.du.is_localhost(self.server.hostname) is True:
+                qmgr_cmd = ["qmgr", "-c", "list sched"]
+            else:
+                qmgr_cmd = ["qmgr", "-c", "\'list sched\'"]
+            with os.fdopen(fd, "w+") as tempfd:
+                ret = self.du.run_cmd(self.server.hostname, qmgr_cmd,
+                                      stdout=tempfd)
+
+                self.assertTrue(ret['rc'] == 0)
+                for line in tempfd:
+                    self.__check_whitespace_prefix(line)
+
+            # Check 2: line extensions are prefixed with tabs and not spaces
+            # Create a random long, comma separated string
+            blah = "blah"
+            long_string = ""
+            for i in range(49):
+                long_string += blah + ","
+            long_string += blah
+            # Create a new vnode
+            attrs = {ATTR_rescavail + ".ncpus": 2}
+            self.server.create_vnodes(node_prefix, attrs, 1, self.mom)
+            # Set 'comment' attribute to the long string we created above
+            attrs = {ATTR_comment: long_string}
+            self.server.manager(MGR_CMD_SET, VNODE, attrs, nodename)
+            # Execute "qmgr 'list node vn[0]'"
+            # The comment attribute should generate a line extension
+            if self.du.is_localhost(self.server.hostname) is True:
+                qmgr_cmd = ["qmgr", "-c", "list node " + nodename]
+            else:
+                qmgr_cmd = ["qmgr", "-c", "\'list node " + nodename + "\'"]
+            with open(fn, "w+") as tempfd:
+                ret = self.du.run_cmd(self.server.hostname, qmgr_cmd,
+                                      stdout=tempfd)
+                self.assertTrue(ret['rc'] == 0)
+                for line in tempfd:
+                    self.__check_whitespace_prefix(line)
+
+        finally:
+            # Cleanup
+            # Remove the temporary file
+            os.remove(fn)
+            # Delete the vnode created
+            if vndef_file is not None:
+                self.mom.delete_vnodes()
+                self.server.manager(MGR_CMD_DELETE, VNODE, id=nodename)
```

Comment:
setUp() is called before every test.  No matter how many tests you run, revert_to_defaults() will be run.  The only thing this will do is delete it prior to leaving the final test.  I'd rather you leave it to revert_to_defaults().  It's the standard place this sort of thing is done in PTL.